packages/metal-incremental-dom/src/incremental-dom.js

```diff
@@ -666,9 +666,7 @@ import { globals } from 'metal';
    * @param {?Object<string, !Element>} keyMap
    */
   var removeChild = function (node, child, keyMap) {
-    if (child.parentNode === node) {
```

Comment:
Just saw in your latest commit, in the repo, looks like they haven't published in the CDN yet.